tendrl/node_agent/manager/alert_socket.py

```diff
@@ -0,0 +1,54 @@
+import SocketServer
+from multiprocessing import Process
+import json
+from tendrl.common.etcdobj.etcdobj import Server as etcd_server
+from tendrl.node_agent.config import TendrlConfig
+import logging
+from tendrl.common.config import ConfigNotFound
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+
+
+class AlertSocketHandler(SocketServer.BaseRequestHandler):
+    def handle(self):
+        try:
+            data = self.request.recv(4096)
```

Comment:
I feel 4096 is size of data to be read. We should define a constant for the same.